charts/rancher-istio/0.0.1/charts/grafana/templates/service.yaml

```diff
@@ -12,21 +12,13 @@ metadata:
     chart: {{ template "grafana.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    kubernetes.io/cluster-service: "true"
```

Comment:
Is there anywhere we use this label?

Reply:
this is required label if we want to use rancher proxy.

Reply:
ok, got it.